src/main/java/it/aboutbits/springboot/toolbox/boot/type/RegisterCustomTypes.java

```diff
@@ -0,0 +1,15 @@
+package it.aboutbits.springboot.toolbox.boot.type;
+
+import org.springframework.context.annotation.Import;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target({ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+@Import(CustomTypeConfigurationRegistrar.class)
+public @interface RegisterCustomTypes {
```

Comment:
I would propose to give this annotation more context, which will make it easier to understand what it is used for in the client application.

```suggestion
public @interface RegisterCustomTypesInJacksonAndMvc {
```